src/util/constants.js

```diff
@@ -1,3 +1,16 @@
+export const STREAM_SORT_OPTIONS = [
```

Comment:
Actually, once this gets towards Mobile, if not inherited in WebViews (which I'm hoping 🤞🏼Then this as well as potentially some/all of our store ! should move into a shared location.